src/js/entities-service/entities/actions.js

```diff
@@ -104,10 +104,18 @@ const _Model = BaseModel.extend({
   },
   canEdit() {
     const currentUser = Radio.request('bootstrap', 'currentUser');
+    const owner = this.getOwner();
```

Comment:
I think even this.. we should call it twice.. once in `if ('work:owned:manage')` and once for team.. there should never be an instance in which both of those flags are true, so we're never saving anything by calling it once, but we do risk not needing to call it.

Reply:
Yup good call, that's been updated.